incubator/node-red/stack.yaml

```diff
@@ -0,0 +1,9 @@
+name: Node-RED
+version: 0.1
```

Comment:
Version field needs to be formatted major.minor.patch here - i.e. should be 0.1.0